agar-mini-map.user.js

```diff
@@ -95,7 +95,14 @@ window.msgpack = this.msgpack;
                         }
                     }
                     mini_map_party.trigger('update-list');
-
+                    break;
+                case 130:
+                    if (agar_server != packet.data.url) {
+                        region_name = $("#region>option[value='"+packet.data.region+"']").text();
```

Comment:
`region_name` and `gamemode_name` should be local variables, would you please add `var` before these variables?